server.js

```diff
@@ -1,11 +1,39 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
+import express from 'express';
+import cors from 'cors';
+import mongoose from 'mongoose';
+import listEndpoints from 'express-list-endpoints';
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || 'mongodb://localhost/happy-thoughts';
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true,
+  },
+  like: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date(),
+  },
+  // createdAt: {
+  //   type: String,
+  //   default: () =>
+  //     new Date(Date.now()).toLocaleString('sv-SE', {
+  //       timeZone: 'Europe/Stockholm',
+  //     }),
+  // },
```

Comment:
I think you already know this, but your time is still showing 2 hours behind in the database. I compared my code with yours, because I´ve used the one that you commented out, checked both backend and frontend and they are the same. Maybe it was something else that crashed it?